### PR TITLE
dotnet format/spellchecking

### PR DESCRIPTION
**Pull request by michaelgsharp**
Ran the `dotnet format` command now that we have the Rosalyn editor config.

Also did various spell checking.

I manually went through the files after the tool ran to make sure nothing looking crazy.

### CODE REVIEW

File: src/Microsoft.ML.Mkl.Redist/Microsoft.ML.Mkl.Redist.csproj [hunk 1]
```diff
@@ -15,7 +15,6 @@
   </PropertyGroup>
   <ItemGroup>
     <Content Include="$(RepoRoot)eng\pkg\CommonPackage.props" Pack="true" PackagePath="build\netstandard2.0\$(MSBuildProjectName).props" />
-    <Content Include="$(RepoRoot)eng\pkg\_._" Pack="true" PackagePath="lib\netstandard2.0\" />
```
Comment (eerhardt) [inline src/Microsoft.ML.Mkl.Redist/Microsoft.ML.Mkl.Redist.csproj:18]: What's the story with this change? It is pretty hidden with a lot of formatting changes. If we really want to make this change, we should make it separate from the rest of these changes.

Comment (michaelgsharp) [reply src/Microsoft.ML.Mkl.Redist/Microsoft.ML.Mkl.Redist.csproj:18]: This is actually a warning you pointed out to me a while ago. We are including the `_._` file in the package, which is supposed to mean the folder is empty, but the folder isn't really empty. So it throws a build warning. I am happy to move it to a new PR though.

**Comment by eerhardt** [comment]
Tests appear to be broken:

```
[37m  Discovering: Microsoft.ML.CodeAnalyzer.Tests (method display = ClassAndMethod, method display options = None)
[m[37m  Discovered:  Microsoft.ML.CodeAnalyzer.Tests (found 19 test cases)
[m[37m  Starting:    Microsoft.ML.CodeAnalyzer.Tests (parallel test collections = on, max threads = 4)
[m[33;1m    Microsoft.ML.CodeAnalyzer.Tests.Code.RelaxTestNamingTest.TestClassWithTheory [SKIP]
[m[37m      https://github.com/dotnet/roslyn/issues/41584
[m[31;1m    Microsoft.ML.InternalCodeAnalyzer.Tests.ContractsCheckTest.ContractsCheckFix [FAIL]
[m[37m      Context: Iterative code fix application
[m[37m      
[m[37m      Expected: True
[m[37m      Actual:   False
[m[30;1m      Stack Trace:
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Testing.Verifiers.XUnit/XUnitVerifier.cs(62,0): at Microsoft.CodeAnalysis.Testing.Verifiers.XUnitVerifier.True(Boolean assert, String message)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(357,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertChildSyntaxListEqual(ChildSyntaxList expected, ChildSyntaxList actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(332,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.Visit(SyntaxNode node)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(337,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertNodesEqual(IVerifier verifier, Type syntaxKindType, SyntaxNode expected, SyntaxNode actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(342,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertNodesEqual(SyntaxNode expected, SyntaxNode actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(358,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertChildSyntaxListEqual(ChildSyntaxList expected, ChildSyntaxList actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(332,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.Visit(SyntaxNode node)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(337,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertNodesEqual(IVerifier verifier, Type syntaxKindType, SyntaxNode expected, SyntaxNode actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(342,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertNodesEqual(SyntaxNode expected, SyntaxNode actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(358,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertChildSyntaxListEqual(ChildSyntaxList expected, ChildSyntaxList actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(332,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.Visit(SyntaxNode node)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(337,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertNodesEqual(IVerifier verifier, Type syntaxKindType, SyntaxNode expected, SyntaxNode actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(342,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertNodesEqual(SyntaxNode expected, SyntaxNode actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(358,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertChildSyntaxListEqual(ChildSyntaxList expected, ChildSyntaxList actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(332,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.Visit(SyntaxNode node)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(337,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertNodesEqual(IVerifier verifier, Type syntaxKindType, SyntaxNode expected, SyntaxNode actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(342,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertNodesEqual(SyntaxNode expected, SyntaxNode actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(358,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertChildSyntaxListEqual(ChildSyntaxList expected, ChildSyntaxList actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(332,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.Visit(SyntaxNode node)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(337,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertNodesEqual(IVerifier verifier, Type syntaxKindType, SyntaxNode expected, SyntaxNode actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(342,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertNodesEqual(SyntaxNode expected, SyntaxNode actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(358,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertChildSyntaxListEqual(ChildSyntaxList expected, ChildSyntaxList actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(332,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.Visit(SyntaxNode node)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(337,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.TreeEqualityVisitor.AssertNodesEqual(IVerifier verifier, Type syntaxKindType, SyntaxNode expected, SyntaxNode actual, Boolean checkTrivia)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(172,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.RecreateProjectDocumentsAsync(Project project, IVerifier verifier, CancellationToken cancellationToken)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(188,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.RecreateProjectDocumentsAsync(Project project, IVerifier verifier, CancellationToken cancellationToken)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/CodeActionTest`1.cs(145,0): at Microsoft.CodeAnalysis.Testing.CodeActionTest`1.ApplyCodeActionAsync(Project project, CodeAction codeAction, IVerifier verifier, CancellationToken cancellationToken)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CodeFix.Testing/CodeFixTest`1.cs(505,0): at Microsoft.CodeAnalysis.Testing.CodeFixTest`1.FixEachAnalyzerDiagnosticAsync(ImmutableArray`1 analyzers, ImmutableArray`1 codeFixProviders, Nullable`1 codeFixIndex, String codeFixEquivalenceKey, Project project, Int32 numberOfIterations, IVerifier verifier, CancellationToken cancellationToken)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CodeFix.Testing/CodeFixTest`1.cs(404,0): at Microsoft.CodeAnalysis.Testing.CodeFixTest`1.VerifyFixAsync(String language, ImmutableArray`1 analyzers, ImmutableArray`1 codeFixProviders, SolutionState oldState, SolutionState newState, Int32 numberOfIterations, Func`9 getFixedProject, IVerifier verifier, CancellationToken cancellationToken)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CodeFix.Testing/CodeFixTest`1.cs(371,0): at Microsoft.CodeAnalysis.Testing.CodeFixTest`1.VerifyFixAsync(SolutionState testState, SolutionState fixedState, SolutionState batchFixedState, IVerifier verifier, CancellationToken cancellationToken)
[m[37m        /_/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CodeFix.Testing/CodeFixTest`1.cs(247,0): at Microsoft.CodeAnalysis.Testing.CodeFixTest`1.RunAsync(CancellationToken cancellationToken)
[m[37m        /Users/runner/work/1/s/test/Microsoft.ML.CodeAnalyzer.Tests/Code/ContractsCheckTest.cs(156,0): at Microsoft.ML.InternalCodeAnalyzer.Tests.ContractsCheckTest.ContractsCheckFix()
[m[37m        --- End of stack trace from previous location where exception was thrown ---
[m[33;1m    Microsoft.ML.CodeAnalyzer.Tests.Code.RelaxTestNamingTest.TestClassWithFact [SKIP]
[m[37m      https://github.com/dotnet/roslyn/issues/41584
[m[37m  Finished:    Microsoft.ML.CodeAnalyzer.Tests
[m[37m=== TEST EXECUTION SUMMARY ===
[m[37m   Microsoft.ML.CodeAnalyzer.Tests  Total: 19, Errors: 0, Failed: 1, Skipped: 2, Time: 19.002s
```

**Comment by michaelgsharp** [comment]
@eerhardt I requeued the tests and that one passed this time. I'll run them all again and see if it fails again, not sure why it would fail first and then pass.

**Review by eerhardt — APPROVED** [review]
Thanks for the work here, @michaelgsharp.

**Pull request merged**